src/hotspot/share/gc/serial/tenuredGeneration.hpp

```diff
@@ -45,7 +45,9 @@ class TenuredGeneration: public Generation {
   // Abstractly, this is a subtype that gets access to protected fields.
   friend class VM_PopulateDumpSharedSpace;
 
- protected:
+  MemRegion _prev_used_region;
+
+protected:
```

Comment:
Some of the GC classes put all variables under a protected section instead of a private section. We just haven't cleaned that up. This means that you should probably stick with the current style to put the variables in the protected section, or better yet, just remove the "protected:" line and let all variables be private.